jupyterlab-dataops/Dockerfile

```diff
@@ -23,9 +22,8 @@ RUN code-server --install-extension ms-python.python@${VSCODE_PYTHON} \
 COPY resources/vsc/settings.json /root/.local/share/code-server/Machine/
 COPY resources/vsc/.bashrc root/
 
-# dbeaver
-COPY --from=builder-dbeaver /opt/cloudbeaver/ /opt/cloudbeaver/
-COPY resources/cloudbeaver/conf /opt/cloudbeaver/conf
+# cloudbeaver
+COPY --from=gcr.io/getindata-images-public/cloudbeaver-dataops /opt/cloudbeaver/ /opt/cloudbeaver/
```

Comment:
My suggestion is to add a tag with a specific version because now I think it will always take the latest.